### ISSUE TITLE
Separation of events

### ISSUE THREAD

**Issue by Luckosta** [llotheo/react-native-big-calendar#904]
Hello, thank you very much for this package. I would like to ask if it is now possible to implement event mapping using this library as follows
<img width="853" alt="image" src="https://github.com/acro5piano/react-native-big-calendar/assets/107638703/d3c6b93b-ebf4-47b4-93ed-13f03709064b">
Thank you!

**Comment by acro5piano** [comment]
I'm not sure what "Event Mapping" means. Can you share additional information?

**Comment by Luckosta** [comment]
Yes, of course, I meant that I do not want events that start at the same time to overlap each other, but to divide the line equally between each other

**Comment by acro5piano** [comment]
Hmm... Can you create screenshots for both **Expected** and **Actual**?

**Comment by Luckosta** [comment]
I want something like this 
![image](https://github.com/acro5piano/react-native-big-calendar/assets/107638703/9b76793a-cc80-4826-b30d-5b3a476484b2)
